### ISSUE TITLE
Recently uploaded packages missing from CDN

### ISSUE THREAD

**Issue by jakirkham** [conda/infra#561]
Noticing some packages are not showing up on CDN. For example

```
$ conda search 'conda-forge::zlib-wapi[subdir=win-64]'
Loading channels: done
No match found for: conda-forge/win-64::zlib-wapi. Search: conda-forge/win-64::*zlib-wapi*

PackagesNotFoundError: The following packages are not available from current channels:

  - conda-forge/win-64::zlib-wapi

Current channels:

  - https://conda.anaconda.org/conda-forge/win-64
  - https://conda.anaconda.org/conda-forge/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

Though can also see this with `libzlib` (even on other platforms). Also taking a recently added package `ncvue` has the same issue.

**Comment by barabo** [comment]
The worker VM that clones the conda-forge channel experienced a double failure - both the root volume and the package volume filled at the same time.  This blocked the clone operation and also suppressed our monitoring.  😦 

I was able to expand both volumes and filesystems, though, and any missing packages should be picked up and propagated within the hour.

**Comment by beckermr** [comment]
We are still missing injector-0.20

```
$ curl -s https://conda.anaconda.org/conda-forge/noarch/repodata.json | jq ".packages|keys" | grep "injector"
  "flask-injector-0.12.3-pyhd8ed1ab_0.tar.bz2",
  "flask-injector-0.13.0-pyhd8ed1ab_0.tar.bz2",
  "injector-0.18.4-pyh44b312d_2.tar.bz2",
  "injector-0.18.4-pyh6c4a22f_2.tar.bz2",
  "injector-0.18.4-pyhd3deb0d_2.tar.bz2",
  "injector-0.18.4-pyhd8ed1ab_3.tar.bz2",
  "injector-0.18.4-pyheb06c22_2.tar.bz2",
  "injector-0.19.0-pyhd8ed1ab_0.tar.bz2",
```

See here: https://anaconda.org/conda-forge/injector/0.20.0/download/noarch/injector-0.20.0-pyhd8ed1ab_0.tar.bz2

**Comment by leofang** [comment]
Still missing libzlib 1.2.12 build 1:
```
$ conda search libzlib
Loading channels: done
# Name                       Version           Build  Channel             
libzlib                       1.2.11   h166bdaf_1014  conda-forge         
libzlib                       1.2.11   h36c2ea0_1012  conda-forge         
libzlib                       1.2.11   h36c2ea0_1013  conda-forge         
libzlib                       1.2.12      h166bdaf_0  conda-forge 
```

**Comment by hmaarrfk** [comment]
from gitter:
![image](https://user-images.githubusercontent.com/90008/173909379-264a30a6-362a-4f38-9bcd-27ccd2f8a189.png)

**Comment by barabo** [comment]
Sorry for the delay, but we've been working hard to find a root cause and we've finally found it and believe that service will be fully restored within the next 30 minutes.

We discovered that when the root volume of the worker VM filled up, it corrupted a few packages that were in the process of being downloaded and it *also* corrupted a local copy of the `repodata.json` that comes from anaconda.org.  This caused the cloning logic to silently fail to parse the cached json - but it did not cause the clone operation to fail.  The effect was that only `linux-64` appeared to be sync'ing and even then there appeared to be nothing for it to sync.

The overall effect of this outage is that any new packages uploaded to conda-forge have not been pushed to the CDN since the volume filled at `2022-06-14T13:48:03Z`.

Now that the cloning service has been restored - the uploaded packages are in the process of landing in the CDN.

**Comment by jakirkham** [comment]
That's great! Thanks for hammering away at this Carl, et. al. 🙏

Any thoughts on how to catch/guard against similar issues in the future?

**Comment by barabo** [comment]
We are in the process of deploying a totally new (and *much* faster) CDN cloner.  The new system won't have this problem - and once it's in place we'll be able to add more monitoring & alerting.  That new system was supposed to be in place last week Friday, but it's been delayed while we iron out a few wrinkles.

We'll make an announcement when we flip the switch.

**Comment by jakirkham** [comment]
Yep heard about the CDN improvements. Am excited to see them 😄 No worries on taking time to get them in place 😉

**Comment by beckermr** [comment]
Oh no! I changed the status page to tighten down the time limits where it says the CDN is broken. I was wondering why the times hadn't gotten shorter. Now I know why!

Thanks for the hard work and good luck!

**Comment by hmaarrfk** [comment]
for what its worth, i didn't mean to ping too much, i was mostly trying to consolidate information for those that didn't check gitter.

**Comment by hmaarrfk** [comment]
thanks for your help!

**Comment by leofang** [comment]
Thanks for the hard work!

**Comment by guyer** [comment]
> i was mostly trying to consolidate information for those that didn't check gitter.

Thank you @hmaarrfk. gitter should be a war crime.
